### PR TITLE
Graph Profile Structure Reformatting

### PR DESCRIPTION
**Pull request by micdavis**
- Reformatted continuous distribution attribute
- Removed unnecessary tests
- Fixed failing tests
- Added `requirements-test.txt`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,12 +42,12 @@ pip install -e .
 Test/Lint Dependencies
 
 ```bash
-$ pip install pandas pytest pytest-cov flake8
+pip install -r requirements-test.txt
```
Comment (micdavis) [inline README.md:45]: consolidated into a file.

File: tests/test_marginal_dist.py [hunk 2]
```diff
@@ -9,10 +9,10 @@
 )
 from scipy import stats
 
-np.random.seed(0)
```
Comment (micdavis) [inline tests/test_marginal_dist.py:12]: tests fail if the seed isn't reset at the beginning of every test

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 numpy>=1.22.0rc1
 scikit-learn==1.0.2
 scipy==1.8.0
-dataprofiler==0.7.10
\ No newline at end of file
+dataprofiler==0.8.0
```
Comment (micdavis) [inline requirements.txt:4]: DP version 0.8.0 is required in order to reflect the changes made to graph profiles.

File: tests/test_graph_synthetic.py [hunk 4]
```diff
@@ -63,49 +65,6 @@ def test_sample_categorical(self):
...
-    
-    def test_plot_sample_categorical(self):
-        np.random.seed(2)
-        attribute = self.synthetic_graph._categorical_attributes[0]
-        data = []
-        for n in range(0, 2000):
-            data.append(self.synthetic_graph.sample_categorical(attribute))
-        
-        hist, edges = np.histogram(data, bins=[1.0, 1.75, 2.5, 3.25, 4.25, 5.25, 6.25, 7.25, 8], density=False)
-        self.assertEqual(list(hist), [24, 45, 374, 379, 201, 81, 72, 824])
```
Comment (JGSweets) [inline tests/test_graph_synthetic.py:75]: there's an assertion here, validity unknown, but calling it out

File: tests/test_graph_synthetic.py [hunk 5]
```diff
@@ -63,49 +65,6 @@ def test_sample_categorical(self):
...
-        ax1.set_title('amplitude-normalized expected distribution')
-        ax2.set_title('amplitude-normalized computed distribution')
-        plt.show()
-
-    def test_plot_sample_continuous(self):
-        np.random.seed(5)
-        attribute = self.synthetic_graph._continuous_attributes[0]
-        data = self.synthetic_graph.sample_continuous(attribute, 2000)
-        properties = self.expected_profile["continuous_distribution"][attribute]["properties"]
-        distribution_continuous_test = st.norm(loc=properties[0], scale=properties[1])
```
Comment (JGSweets) [inline tests/test_graph_synthetic.py:99]: should this still exist, but do the same assertion as above?
running the func should still be validated: `data = self.synthetic_graph.sample_continuous(attribute, 2000)`

File: tests/test_graph_synthetic.py [hunk 6]
```diff
@@ -63,49 +66,23 @@ def test_sample_categorical(self):
...
-        bin_lims = np.linspace(0,1,num_bin+1)
-        bin_centers = 0.5*(bin_lims[:-1]+bin_lims[1:])
-        bin_widths = bin_lims[1:]-bin_lims[:-1]
-
-        fig, (ax1,ax2) = plt.subplots(nrows = 1, ncols = 2)
-        ax1.bar(bin_centers, hist, width = bin_widths, align = 'center')
-        ax2.bar(bin_centers, expected_hist, width = bin_widths, align = 'center', alpha = 0.5)
-        ax1.set_title('amplitude-normalized expected distribution')
-        ax2.set_title('amplitude-normalized computed distribution')
-        plt.show()
```
Comment (ssharpe42) [inline tests/test_graph_synthetic.py:92]: Does the plot showing essentially cause the tests to hang?

**Review by ssharpe42 — APPROVED** [review]

**Pull request merged**